apps/web/components/booking/CancelBooking.tsx

```diff
@@ -1,6 +1,6 @@
-import { useRouter } from "next/navigation";
-import { useCallback, useState } from "react";
 
+import { useRouter } from "next/router";
```

Comment:
Can you try importing useRouter from `next/navigation` here?

Reply:
yes doing it